### ISSUE TITLE
New card charts

### ISSUE THREAD

**Issue by andersonjeccel** [mautic/mautic#13168]
### Description
The current card charts lack modern design and functionality, limiting their effectiveness in presenting data; we aim to introduce new card charts with improved aesthetics and features.

We have to deal with chartjs on this task.

### Impact
Implementing new card charts will enhance data visualization, provide better insights, and improve the overall user experience, making it easier for users to interpret and interact with data displayed in the application.

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by Vayras** [comment]
Hey! @andersonjeccel I'm a open source developer I came across this issue ,I would like to contribute towards this , If you can provide me with the expected design and the old design I can start working on this ,Thanks

**Comment by andersonjeccel** [comment]
Hey @Vayras, thanks for your interest.

Here is the old design:

![Image](https://github.com/mautic/mautic/assets/116097999/a012a987-2e45-4329-ac10-b45a0c9efa07)


We would like to see something cleaner, less grid lines. We like these references:

![Image](https://github.com/mautic/mautic/assets/116097999/627f61f0-017b-4509-819b-94eca115c0b0)


![Image](https://github.com/mautic/mautic/assets/116097999/80cbf761-d0dc-4065-b094-69ec8818170f)

A future plan is to add information like "Total leads" (As the example of Income or Transactions on the first reference) in a card that contains only number and titles without graphs. Let me know if you'd like to add such backend functionality so we can add them later.
